astropy/wcs/tests/test_keyedsip.py

```diff
@@ -0,0 +1,17 @@
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ... import wcs
+
+def read_keyed_wcs():
+    w = wcs.WCS( header = "data/sip_a_only.hdr", key = 'A' )
+    return( w )
+
+def test_sip():
+    w = read_keyed_wcs()
+    print( w.sip.crpix )
+    assert isinstance( w.sip, wcs.Sip )
+    assert w.sip.crpix[0] > 0
+    assert w.sip.crpix[1] > 0
+
```

Comment:
Please check for specific values instead of using inequality.

Also could you add the test to test_wcs.py.